src/pages/progress_reports/SummarySection.tsx

```diff
@@ -1,18 +1,24 @@
-import React, { useState } from 'react';
-import { FORM_TEXT_AREA_MAX_LENGTH } from 'src/components/forms/formConstants';
-import { FormField, FormFieldTypes } from 'src/components/forms/forms';
-import { Publication } from 'src/components/publications_list/Publication';
+import React, {useState} from 'react';
+import {FORM_TEXT_AREA_MAX_LENGTH} from 'src/components/forms/formConstants';
+import {FormField, FormFieldTypes} from 'src/components/forms/forms';
+import {Publication} from 'src/components/publications_list/Publication';
 import PublicationList from 'src/components/publications_list/PublicationList';
-import { FormFieldChange, FormState } from 'src/pages/progress_reports/ProgressReportFormState';
+import {FormFieldChange, FormState} from 'src/pages/progress_reports/ProgressReportFormState';
+import ERACommons from 'src/components/ERACommons';
+import {DuosUser} from 'src/types/model';
+import {Location} from 'history';
 
 interface SummarySectionProps {
     readonly readOnly: boolean;
     formState: FormState;
     onFormChange: (newState: Partial<FormState>) => void;
+    readonly eRACommonsDestination?: string
+    readonly location?: Location
```

Comment:
Why are these optional?

Reply:
`eRACommonsDestination` is only necessary when being redirected away from DUOS to NIH for authentication, so we don't need it in read-only mode. Same for `location`.

Reply:
ditto my previous comment